### ISSUE TITLE
Test failure in JIT/Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#33138]
failed in job: [runtime-coreclr outerloop 20200303.6](https://dev.azure.com/dnceng/public/_build/results?buildId=545440&view=ms.vss-test-web.build-test-results-tab&runId=17260168&resultId=105964&paneView=debug)

Error message:
`
\nAssert failure(PID 8876 [0x000022ac], Thread: 8876 [0x22ac]): Assertion failed '!"PerfScore: unhandled instruction"' in 'test:Main():int' during 'Emit code' (IL size 267730)\n\n File: /__w/6/s/src/coreclr/src/jit/emit.cpp Line: 1145\n Image: /root/helix/work/correlation/crossgen\n\n/root/helix/work/workitem/JIT/Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.sh: line 157: 8876 Aborted __Command=$_DebuggerFullPath "$CORE_ROOT/crossgen" /Platform_Assemblies_Paths $CORE_ROOT:$PWD /in b28158.org /out b28158.dll\nManaged assembly not found: No such file or directory\n\nReturn code: 1\nRaw output file: /root/helix/work/workitem/JIT/Regression/Reports/JIT.Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.output.txt\nRaw output:\nBEGIN EXECUTION\nin takeLock\nMicrosoft (R) CoreCLR Native Image Generator - Version 4.5.30319.0\nCopyright (c) Microsoft Corporation. All rights reserved.\n\nPerfScore: unhandled instruction: adr, format IF_LARGEADR\nin ReleaseLock\nin takeLock\n/root/helix/work/correlation/corerun b28158.dll ''\nin ReleaseLock\nExpected: 100\nActual: 255\nEND EXECUTION - FAILED\nTest Harness Exitcode is : 1\nTo run the test:\n> set CORE_ROOT=/root/helix/work/correlation\n> /root/helix/work/workitem/JIT/Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.sh\nExpected: True\nActual: False
`

**Comment by BruceForstall** [comment]
Failed in "R2R Linux arm64 Checked no_tiered_compilation"

**Comment by BruceForstall** [comment]
cc @echesakovMSFT

**Comment by danmosemsft** [comment]
newlines fixed:
```

Assert failure(PID 8876 [0x000022ac], Thread: 8876 [0x22ac]): Assertion failed '!"PerfScore: unhandled instruction"' in 'test:Main():int' during 'Emit code' (IL size 267730)

 File: /__w/6/s/src/coreclr/src/jit/emit.cpp Line: 1145
 Image: /root/helix/work/correlation/crossgen

/root/helix/work/workitem/JIT/Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.sh: line 157: 8876 Aborted __Command=$_DebuggerFullPath "$CORE_ROOT/crossgen" /Platform_Assemblies_Paths $CORE_ROOT:$PWD /in b28158.org /out b28158.dll
Managed assembly not found: No such file or directory

Return code: 1
Raw output file: /root/helix/work/workitem/JIT/Regression/Reports/JIT.Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.output.txt
Raw output:
BEGIN EXECUTION
in takeLock
Microsoft (R) CoreCLR Native Image Generator - Version 4.5.30319.0
Copyright (c) Microsoft Corporation. All rights reserved.

PerfScore: unhandled instruction: adr, format IF_LARGEADR
in ReleaseLock
in takeLock
/root/helix/work/correlation/corerun b28158.dll ''
in ReleaseLock
Expected: 100
Actual: 255
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/root/helix/work/correlation
> /root/helix/work/workitem/JIT/Regression/VS-ia64-JIT/V1.2-M02/b28158/b28158/b28158.sh
Expected: True
Actual: False
```

**Comment by briansull** [comment]
Fixed with  #33161
